_includes/tutorials/creating-first-apache-kafka-streams-application/kstreams/markup/dev/answer-short.adoc

```diff
@@ -0,0 +1,5 @@
+Using the Apache Kafka Streams DSL, create a stream processing topology to define your business logic. The example below reads events from the input topic using the `stream` function, processes events using the `mapValues` transformation, and writes the transformed events to an output topic using `to`.
+
```

Comment:
Because the example also shows `peek()`, is it worth a note on that?

Reply:
Added blurb in 7d248cbb